test/performance/bpmn.navigation.performance.test.ts

```diff
@@ -48,14 +48,14 @@ describe.each([1, 2, 3, 4, 5])('zoom performance', run => {
     const metricsStart = await metricsCollector.metrics();
 
     for (let i = 0; i < xTimes; i++) {
```

Comment:
It's the same than `doChromiumMouseZoom` from `test/e2e/helpers/test-utils.ts`, but with another delay.
Is it possible to use this method by setting the delay ?